scripts/get-20newsgroup-data.sh

```diff
@@ -0,0 +1,8 @@
+FOLDER=20-newsgroup
```

Comment:
hrm, I was thinking of `bin/` for shell scripts and `scripts/` for end-to-end Python scripts... but open to discussion?

Reply:
Ok, I'm fine with that.


I will make the following changes:
- bin/get-20newsgroup-data.sh
- scripts/generate_trec_covid_round1_OSHU_RUN2.py
- scripts/20newsgroup-replication.py

Reply:
Done